src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -17,7 +17,7 @@
 public class ManufacturerDaoImpl implements ManufacturerDao {
     @Override
     public Manufacturer create(Manufacturer manufacturer) {
-        String query = "INSERT INTO manufacturers (name, country) "
+        String query = "INSERT INTO manufactures (name, country) "
```

Comment:
```suggestion
        String query = "INSERT INTO manufacturers (name, country) "
```
return to the previous version for all queries